src/UserService.Business/Commands/Certificate/CreateCertificateCommand.cs

```diff
@@ -80,7 +79,7 @@ public class CreateCertificateCommand : ICreateCertificateCommand
             IDbUserCertificateMapper mapper,
             IUserRepository userRepository,
             ICertificateRepository certificateRepository,
-            IRequestClient<IAddImageRequest> rcAddIImage,
+            IRequestClient<ICreateImagesRequest> rcAddIImage,
```

Comment:
rcAddIImage => rcImage

Reply:
why not rcAddImage?

Reply:
you have above  IRequestClient<ICreateImagesRequest> _rcImage, 
and in other classes everywhere 'rcImage'

Reply:
Certificates will be rewrited by Ilya